docs/source/using/accessing-the-openshift-docker-registry.adoc

```diff
@@ -0,0 +1,44 @@
+= Accessing the OpenShift Docker Registry
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+OpenShift provides an integrated docker registry which can be used as a local registry.
```

Comment:
Not sure how to reword this. This supposed to communicate the user that the docker registry provided by OpenShift through Minishift can used as a docker registry which is local to the workstation i.e. user has a local docker registry to push docker images.